### DISCUSSION TITLE
Option to Add Plus and Minus Button in Add to cart button

### DISCUSSION THREAD

**Discussion by MdRasheid** [elementor/elementor#23846] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When using a taxonomy filter with an astra theme enabled with woocommerce misc plus and minus button, After selecting filter the misc Plus and Minus buttons gets removed

Steps to reproduce
In the Astra theme enable the Misc plus and minus button
Create loop grid for products with elementor
Add a Taxonomy filter in the products page for filtering loop grid
If we click any category for filter plus and minus buttons gets removed

### Describe the solution you'd like

Add an option in add to cart widget for plus and minus button

**Comment by jazir555** [comment]
I definitely have had this same issue for a long time now. Would be great to see it fixed.
